app/models/claim/advocate_claim.rb

```diff
@@ -232,12 +236,15 @@ def set_supplier_number
 
     # create a blank fee for every basic fee type not passed to Claim::AdvocateClaim.new
     def instantiate_basic_fees
-      return unless new_record?
+      return unless case_type.present? && !case_type.is_fixed_fee?
```

Comment:
This method has been creating all sort of issues/complications throughout the development process. I've decided to keep it for now as I still lack full understanding of the impact of removing it and managing what is trying to achieve elsewhere.

Still, because of that, to ensure the management of the **basic fees** vs **fixed fees** and **basic fees for scheme 9** vs **basic fees for scheme 10**, I had to make some changes to it to ensure that behaviour was still working.